app/index.html

```diff
@@ -7,7 +7,7 @@
     <meta content="IE=edge" http-equiv="X-UA-Compatible">
     <link rel="icon" href="favicon.ico" type="image/x-icon">
     <link rel="stylesheet" href="/css/style.css">
-    <link rel="stylesheet" href="/bower_components/angular-chosen-localytics/node_modules/chosen-js/chosen.css">
+    <link rel="stylesheet" href="/css/chosen.css">
```

Comment:
I think this file is included somewhere in the `bower_components` imports. For future-proofing sakes it may be better to find where it is in there and reference it rather than including it seperatly in `css`